DataFormats/Detectors/TRD/include/DataFormatsTRD/Digit.h

```diff
@@ -38,15 +38,28 @@ using ArrayADC = std::array<ADC_t, constants::TIMEBINS>;
 //        this negates the need for need alternate indexing strategies.
 //        if you are trying to go from mcm/rob/adc to pad/row and back to mcm/rob/adc ,
 //        you may not end up in the same place, you need to remember to manually check for shared pads.
+//    Pre Trigger phase:
+//        LHC clock runs at 40.08MHz, ADC run at 1/4 or 10MHz and the trap runs at 120MHz or LHC*3.
+//        The phase then has 4 values although it has 4 bits, which is rather confusing.
```

Comment:
which one is it now, 4 or 12 values?

Reply:
4 values as per drawing, where do you see 12 written?

Reply:
It *can* have one of 12 values as per drawing, but only 4 are valid.